src/main/app/case/from-api-format.ts

```diff
@@ -19,6 +28,32 @@ const checkboxConverter = (value: string | undefined) => {
   return value === YesOrNo.YES ? Checkbox.Checked : Checkbox.Unchecked;
 };
 
+const marriageBrokenConverter = data => {
```

Comment:
do we need two converters here? could we not combine them like you do in the to-api, given that it's either DivorceOrDissolution.DIVORCE or DivorceOrDissolution.DISSOLUTION

Reply:
From my understanding as there are 2 API fields, each one will call the same conversion which will end up with the same function being called twice if it's combined, which could change the value depending on the order it's called?

Reply:
I think the values would still match, no?